deeprankcore/trainer.py

```diff
@@ -518,8 +518,9 @@ def train( # pylint: disable=too-many-arguments, too-many-branches, too-many-loc
         min_epoch: int = 10, 
         validate: bool = False,
         num_workers: int = 0,
-        save_best_model: Optional[bool] = True,
-        output_prefix: Optional[str] = None,
+        best_model: bool = True,
+        save_model: bool = True,
+        filename: str = 'model.pth.tar'
```

Comment:
How about reducing the number of parameters by allowing the filename to be optional and if None --> don't save?


```suggestion
        filename: Optional[str] = 'model.pth.tar'
        best_model: bool = True,
```

Reply:
I think you need it in any case. It determines which model is going to be used for later testing (last or best), even if the model itself is not saved in a file. 

I can remove save_model and make filename optional though.